### PR TITLE
Added data collection component

### PR DESCRIPTION
**Pull request by nwkotto**
This component introduces radio button cards, meant to be included in the onboarding flow, for setting who decides where data can be collected.

### CODE REVIEW

**Review by os-micmec — DISMISSED** [review]

File: web/src/app/pages/create-survey/data-collection/data-collection.module.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    CommonModule,
+    MatButtonModule,
+    MatCardModule,
+    MatFormFieldModule,
+    MatRadioModule,
+    ReactiveFormsModule,
+  ],
+  exports: [DataCollectionComponent],
+})
+export class DataCollectionModule {}
```
Comment (gino-m) [inline web/src/app/pages/create-survey/data-collection/data-collection.module.ts:38]: Nit: Data collection in the context of Ground refers to the actual entry of field data by data collections. Can we rename this to something self-descriptive that doesn't collide with that name? 

In plain-English: "This component allows survey organizers to select who can specify data collection sites (i.e., locations of interest for which data is to be collected)." So maybe something like "LoiPermissionsComponent"?

**Review by gino-m — APPROVED** [review]

**Review by gino-m — CHANGES_REQUESTED** [review]
One nit, otherwise LG!

**Review by DaoyuT — APPROVED** [review]
LGTM

**Comment by DaoyuT** [comment]
@gino-m please stamp again if you think that one nit has been addressed :^)

**Review by gino-m — APPROVED** [review]
LGTM, thanks @nwkotto !

**Review by gino-m — APPROVED** [review]

**Pull request merged**